test/Makefile

```diff
@@ -41,3 +41,9 @@ module: deps
 examples/complete: export TESTS ?= installed lint get-modules get-plugins validate
 examples/complete: deps
 	$(call RUN_TESTS, ../$@)
+
+## Run tests against config module
```

Comment:
@osterman I added a separate test here just for `config` -- let me know if that's what you had in mind.

Also, the tests are passing, although it doesn't look like `terratest` is really doing all that much when I inspect the action logs, so I figured I should check with you on that one.